src/lib/components/bottom-nav-settings.svelte

```diff
@@ -0,0 +1,88 @@
+<script lang="ts">
+  import { onDestroy } from 'svelte';
+
+  import DataEncoderSettings from '$lib/components/data-encoder-settings.svelte';
+  import TimezoneSelect from '$lib/components/timezone-select.svelte';
+  import NavigationButton from '$lib/holocene/navigation/navigation-button.svelte';
+  import { translate } from '$lib/i18n/translate';
+  import { authUser } from '$lib/stores/auth-user';
+  import { dataEncoder } from '$lib/stores/data-encoder';
+  import { labsMode } from '$lib/stores/labs-mode';
+  import { goto } from '$lib/svelte-mocks/app/navigation';
+  import { useDarkMode } from '$lib/utilities/dark-mode';
+
+  import { viewDataEncoderSettings } from './data-encoder-settings.svelte';
+
+  export let open = false;
+  export let logout: () => void;
+  export let userEmaiLink = '';
+
+  $: labsHoverText = `${translate('common.labs')} ${
+    $labsMode
+      ? `${translate('common.on')} - ${translate('common.experimental')}`
+      : translate('common.off')
+  }`;
+  $: labsText = `${translate('common.labs')} ${
+    $labsMode ? translate('common.on') : translate('common.off')
+  }`;
+
+  $: hasCodecServer = $dataEncoder?.endpoint;
+
+  const onCodecServerClick = () => {
+    $viewDataEncoderSettings = !$viewDataEncoderSettings;
+  };
+
+  onDestroy(() => {
+    $viewDataEncoderSettings = false;
+  });
+</script>
+
+{#if open}
+  <div class="flex h-full flex-col justify-start gap-4">
+    <TimezoneSelect position="left" />
+    {#if $dataEncoder.hasError}
+      <p class="text-red-400">{translate('data-encoder.codec-server-error')}</p>
+    {/if}
+    <NavigationButton
+      onClick={onCodecServerClick}
+      tooltip={translate('data-encoder.codec-server')}
+      label={translate('data-encoder.codec-server')}
+      icon={hasCodecServer ? 'transcoder-on' : 'transcoder-off'}
+    />
+    <DataEncoderSettings />
+    <div class="border-b-2 border-subtle" />
+    <NavigationButton
+      onClick={() => ($useDarkMode = !$useDarkMode)}
+      tooltip={$useDarkMode
+        ? translate('common.night')
+        : translate('common.day')}
+      label={$useDarkMode ? translate('common.night') : translate('common.day')}
+      icon={$useDarkMode ? 'moon' : 'sun'}
+    />
+    <NavigationButton
+      onClick={() => ($labsMode = !$labsMode)}
+      tooltip={labsHoverText}
+      label={labsText}
+      icon="labs"
+      active={$labsMode}
+      data-testid="labs-mode-button"
+    />
+    {#if $authUser.accessToken}
+      <div class="border-b-2 border-subtle" />
+      <NavigationButton
+        onClick={() => userEmaiLink && goto(userEmaiLink)}
```

Comment:
`NavigationButton` should probably accept an `href` so that we can render an anchor tag here, or can we use `NavigationItem` ?

Reply:
👍 I should use NavigationItem